src/server/services/procedures/smithsonian/smithsonian.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * The Smithsonian Service provides access to the Smithsonan open-access EDAN database,
+ * containing catalogued information on millions of historical items.
+ * 
+ * @service
+ */
+'use strict';
+
+const ApiConsumer = require('../utils/api-consumer');
+const {SmithsonianKey} = require('../utils/api-key');
+const Smithsonian = new ApiConsumer('Smithsonian', 'https://api.si.edu/openaccess/api/v1.0', {cache: {ttl: 5*60}});
+ApiConsumer.setRequiredApiKey(Smithsonian, SmithsonianKey);
+
+/**
+ * Searches the EDAN db and returns up to count ids denoting matching historical objects
+ * 
+ * @param {String} term Term to search for
+ * @param {BoundedNumber<0>} start Starting index of search (default 0)
```

Comment:
Makes sense. Optional arguments are supported by NetsBlox RPCs so that isn't an issue.

Yeah, it isn't obvious what the best approach is for these sorts of parameters. Do you know how many results are returned generally? I lean toward keeping them but maybe increasing the limit to something like 100.

Reply:
Also, the limit should correspond to the number of results returned to the user - not the limit before filtering the results.

Reply:
Yeah, I agree on the returned to user vs before filter point, but as far as I can see there's not really a way to do that neatly given that it would require an arbitrary number of API calls due to them not supporting it at the API level. Your point about having both filter and non-filter versions could solve that, though. As long as it's not confusing at a user level that images aren't guaranteed to exist unless using the filtering search.